### ISSUE TITLE
Display text on multiple lines

### ISSUE THREAD

**Issue by pixelplant** [iqnivek/react-circular-progressbar#59]
I was trying to customise the "textForPercentage" function and pass it a newline (```<br/>```) tag but html tags get removed.
Is there an easy way to keep the percentages displayed (and animated) and just display a text below the percentage?
Like in this picture
<img  alt="screencap" src="https://user-images.githubusercontent.com/492082/41645230-a16ffc1e-7470-11e8-8053-e0060613301e.png">

This is how I tried to call textForPercentage:
```
textForPercentage={(p) => { return `${p}%<br/>sold` }}
```
If html is not desired, maybe just support the "\n" newline character and replace it with a ```<br/>``` on display?
So
```
textForPercentage={(p) => { return `${p}%\nsold` }}
```
would also be a valid option

**Comment by pixelplant** [comment]
maybe splitting every "\n" inside textForPercentage into <tspan>s? that use the same CENTER_X and an offset CENTER_Y with a new option called "textLineOffset" that could allow us to specify a specific y offset for tspans?

https://stackoverflow.com/questions/31469134/how-to-display-multiple-lines-of-text-in-svg

**Comment by iqnivek** [comment]
That approach seems possible, but I think a pure svg approach is tricky in general. The approach I've recommended is wrapping the base CircularProgressbar in another component that overlays regular html content on top of it with absolute positioning. You can try it out here: https://codesandbox.io/s/qlr7w0rm29

Let me know if that would address your needs!

**Comment by pixelplant** [comment]
@iqnivek Yes, since it gives you quite a bit of freedom on what you can add inside the progressbar, your example is a good starting point and with a bit of modification it should address my needs. Maybe it would be a good thing to add it to the README on the homepage as I assume this is quite a frequent request from integrators.

Most people would like to use it though with at least 2 lines of text (percentage or count + small description on the second line) so if I find the time I'll try to fork it and work an the <tspan> option

Thanks

**Comment by iqnivek** [comment]
Yup that's a good callout - I'm planning to document this better in the README and probably also export the wrapper component as part of the npm package itself. You're very welcome to fork though!

I'll keep this issue open as a reminder until I add better documentation. Thanks @pixelplant.

**Comment by benfluleck** [comment]
Adding to this conversation, Is it compulsory the text prop is tied to PropType string? For example, passing text through a translation library would render a Node element which would be rendered like a string but would give a PropType warning for this library on the browser.

**Comment by benfluleck** [comment]
I saw the really good solution you used to position images inside the circle, however, this is only effective if the progress circle is staying the same size through out the app.
